File: packages/expo-app-auth/android/src/main/java/expo/modules/appauth/AppAuthConstants.java
package expo.modules.appauth;

public class AppAuthConstants {
  static String MANIFEST_URL = "experienceUrl";

  public interface Error {
    String DEFAULT = "ERR_APP_AUTH";
    String CONCURRENT_TASK = "ERR_APP_AUTH_CONCURRENT_TASK";
  }

  public interface Props {
    String TOKEN_ENDPOINT = "tokenEndpoint";
    String AUTHORIZATION_ENDPOINT = "authorizationEndpoint";
    String REGISTRATION_ENDPOINT = "registrationEndpoint";
    String ISSUER = "issuer";
    String REDIRECT_URL = "redirectUrl";
    String CLIENT_ID = "clientId";
    String CLIENT_SECRET = "clientSecret";
    String REFRESH_TOKEN = "refreshToken";
    String CAN_MAKE_INSECURE_REQUESTS = "canMakeInsecureRequests";
    String IS_REFRESH = "isRefresh";
    String ADDITIONAL_PARAMETERS = "additionalParameters";
    String SCOPES = "scopes";
    String SERVICE_CONFIGURATION = "serviceConfiguration";
    String ID_TOKEN = "idToken";
    String ACCESS_TOKEN_EXPIRATION_DATE = "accessTokenExpirationDate";
    String ACCESS_TOKEN = "accessToken";
    String TOKEN_TYPE = "tokenType";
  }

  public interface HTTPS {
    String CLIENT_SECRET = "client_secret";
    String DISPLAY = "display";
    String PROMPT = "prompt";
    String LOGIN_HINT = "login_hint";
    String NONCE = "nonce";
  }
}

File: packages/expo-app-auth/android/src/main/java/expo/modules/appauth/AppAuthConstants.kt
package expo.modules.appauth

object AppAuthConstants {
  @JvmField
  var MANIFEST_URL = "experienceUrl"

  interface Error {
    companion object {
      const val DEFAULT = "ERR_APP_AUTH"
      const val CONCURRENT_TASK = "ERR_APP_AUTH_CONCURRENT_TASK"
    }
  }

  interface Props {
    companion object {
      const val TOKEN_ENDPOINT = "tokenEndpoint"
      const val AUTHORIZATION_ENDPOINT = "authorizationEndpoint"
      const val REGISTRATION_ENDPOINT = "registrationEndpoint"
      const val ISSUER = "issuer"
      const val REDIRECT_URL = "redirectUrl"
      const val CLIENT_ID = "clientId"
      const val CLIENT_SECRET = "clientSecret"
      const val REFRESH_TOKEN = "refreshToken"
      const val CAN_MAKE_INSECURE_REQUESTS = "canMakeInsecureRequests"
      const val IS_REFRESH = "isRefresh"
      const val ADDITIONAL_PARAMETERS = "additionalParameters"
      const val SCOPES = "scopes"
      const val SERVICE_CONFIGURATION = "serviceConfiguration"
      const val ID_TOKEN = "idToken"
      const val ACCESS_TOKEN_EXPIRATION_DATE = "accessTokenExpirationDate"
      const val ACCESS_TOKEN = "accessToken"
      const val TOKEN_TYPE = "tokenType"
    }
  }

  interface HTTPS {
    companion object {
      const val CLIENT_SECRET = "client_secret"
      const val DISPLAY = "display"
      const val PROMPT = "prompt"
      const val LOGIN_HINT = "login_hint"
      const val NONCE = "nonce"
    }
  }
}